tests/.netrc

```diff
@@ -0,0 +1,3 @@
+machine test_entrc.httpx.com
```

Comment:
Let's use the `example.org` domain, since it's there for this sort of thing.

Reply:
If appropriate, I used netrcexample.org. Because some tests are affected by example.org.